keps/sig-scheduling/20180409-scheduling-framework.md

```diff
@@ -703,4 +703,4 @@ End-to-end tests are not needed when integration tests can provided adequate cov
 # IMPLEMENTATION HISTORY
 
 TODO: write down milestones and target releases, and a plan for how we will
-gracefully move to the new system
\ No newline at end of file
+gracefully move to the new system
```

Comment:
what's the change here?

Reply:
@Huang-Wei I have no idea, actually the content is not change, I think it's caused by `git` itself : )